LoopKit/GlucoseRangeSchedule.swift

```diff
@@ -88,6 +91,10 @@ public struct GlucoseRangeSchedule: DailySchedule {
         public func isActive(at date: Date = Date()) -> Bool {
             return activeDates.contains(date) && !value.isZero
         }
+
+        public var quantityRange: Range<HKQuantity> {
```

Comment:
ClosedRange is a better fit than Range here; if a user's target range is 100-100, we want to represent it as a range containing the single value 100. Using a half-open range would imply an empty range.